efel/cppcore/LibV3.cpp

```diff
@@ -1004,7 +1005,8 @@ static int __AP_fall_indices(const vector<double>& v, const vector<int>& apbi,
     vector<double> vpeak(&v[pi[i]], &v[apei[i]]);
     transform(vpeak.begin(), vpeak.end(), vpeak.begin(),
               bind2nd(std::minus<double>(), halfheight));
-    transform(vpeak.begin(), vpeak.end(), vpeak.begin(), std::ptr_fun(::fabs));
+    transform(vpeak.begin(), vpeak.end(), vpeak.begin(), 
+              static_cast<double(*)(double)>(fabs));
```

Comment:
What's apparently happening is that ::fabs is overloaded a couple of times for floats, doubles, etc. and that the compiler couldn't decide which one to pick